src/constants.js

```diff
@@ -0,0 +1,9 @@
+export const firebaseConfig = {
+  apiKey: "AIzaSyCQhBv01Inz9eX7bC52TJnO3KyZ0odg6b4",
```

Comment:
Looks like you're exposing your secrets inside a public git repo :0

Reply:
According to the [Firebase docs](https://firebase.google.com/docs/web/setup?authuser=0#config-object), "The Firebase config object contains unique, but non-secret identifiers for your Firebase project." It seems like the apiKey is just a badly-namely unique but non-secret identifier. Security in accessing the databsae is handled by the security rules. This is also confirmed by this [Stack Overflow question](https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public)